modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -282,6 +282,189 @@ enum Breakpoint {
+  observed.
+  """
+  activeStart: Timestamp!
+
+  """
+  The end time (exclusive) when the observations accepted for this Call may be
+  observed.
+  """
+  activeEnd: Timestamp!
+
+  """
+  Partners that may participate in this Call.
+  """
+  partners: [CallForProposalsPartner!]!
+
+  """
+  When specified, the observations executed in this Call will only use these
+  instruments.  When not specified, all otherwise available instruments may be
+  used.
+  """
+  instruments: [Instrument!]!
+
+  """
+  Whether this Call is PRESENT or has been DELETED.
+  """
+  existence: Existence!
+}
+
+"""
+Defines a unique ID for each Call for Proposals.
+"""
+scalar CallForProposalsId
+
+"""
+The properties of a Call for Proposal in an input for creation and editing.
+"""
+input CallForProposalsPropertiesInput {
+
+  "Open/closed status.  On create defaults to CLOSED."
+  status: CallForProposalsStatus
+
+  "Type of the call. Required on create."
+  type: CallForProposalsType
+
+  "Semester associated with the call. Required on create."
+  semester: Semester
+
+  "Optional RA limit start RA. Required only if raLimitEnd is specified."
+  raLimitStart: RightAscensionInput
+
+  "Optional RA limit end RA. Required only if raLimitStart is specified."
+  raLimitEnd: RightAscensionInput
+
+  """
+  Optional declination limit start declination. Required only if decLimitEnd is
+  specified.
+  """
+  decLimitStart: DeclinationInput
+
+  """
+  Optional declination limit end declination. Required only if decLimitStart is
+  specified.
+  """
+  decLimitEnd: DeclinationInput
+
+  """
+  Active period start time for this call.  Required on create.  Must be before
+  the activeEnd time.
+  """
+  activeStart: Timestamp
+
+  """
+  Active period end time for this call.  Required on create.  Must be after
+  the activeStart time.
+  """
+  activeEnd: Timestamp
+
+  """
+  Partners that may participate in the call along with their respective
+  deadlines.
+  """
+  partners: [CallForProposalsPartnerInput!]
+
+  """
+  When specified, the call is limited to the listed instruments.  When not
+  specified, all otherwise available instruments may be used.
+  """
+  instruments: [Instrument!]
+
+  "DELETED or PRESENT.  On create defaults to PRESENT."
+  existence: Existence
+}
+
+"""
+Groups a partner with its submission deadline.
+"""
+type CallForProposalsPartner {
+  partner:  Partner!
+  deadline: Timestamp!
+}
+
+input CallForProposalsPartnerInput {
+  partner:  Partner!
+  deadline: Timestamp!
+}
+
+type CallsForProposalsSelectResult {
+  matches: [CallForProposals!]!
+  hasMore: Boolean!
+}
+
+"""
+A CallForProposals may be OPEN or CLOSED.  Submissions are only accepted when
+OPEN regardless of each partner's submission deadline.
+"""
+enum CallForProposalsStatus {
+  OPEN
+  CLOSED
+}
+
+enum CallForProposalsType {
+  DEMO_SCIENCE
+  DIRECTORS_TIME
+  FAST_TURNAROUND
+  LARGE_PROGRAM
+  POOR_WEATHER
+  REGULAR_SEMESTER
```

Comment:
Are Exchanges inside Regular?

Reply:
Good question.  It's unclear what we'll need to do for exchange time and they are missing from the CfP document that I based this on.